scripts/deploy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/zsh
+
+WD=$(readlink -f $(dirname "$0"))
+
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+function win() {
+    printf "\n${GREEN}$1${NC}\n"
+}
+
+function warn() {
+    printf "\n${YELLOW}$1${NC}\n"
+}
+
+function error() {
+    printf "\n${RED}$1${NC}\n"
+}
+
+
+pushd "${WD}/.."
```

Comment:
I usually pipe these to /dev/null. The output isn't very useful in the context of a script.